<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Explorer</class>
 <widget class="QWidget" name="Explorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileExplorer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QStackedWidget" name="stackedWidget"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelGrouping">
         <property name="text">
          <string>Grouping:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="groupBox">
         <item>
          <property name="text">
           <string>By Folders</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>By Types</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDisplay">
         <property name="text">
          <string>Display:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="displayBox">
         <item>
          <property name="text">
           <string>In Table</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In Pie</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In Bar</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
